pyproject.toml

```diff
@@ -41,3 +41,8 @@ output-format = "full"
 
 [tool.ruff.lint]
 select = ["E9", "F63", "F7", "F82"]
+ignore = ["E731"]
```

Comment:
What does this have to do with djLint?

Reply:
It doesn't. This is a leftover from when it was based on a different version of main and I incorrectly rebased it, will fix this now